Spanner/src/Instance.php

```diff
@@ -145,7 +158,8 @@ public function __construct(
         $projectId,
         $name,
         $returnInt64AsObject = false,
-        array $info = []
+        array $info = [],
```

Comment:
Note:
`$info` is a representation of the `instance` and it has its own responsibility. See [here](https://github.com/googleapis/google-cloud-php/blob/8c36f3cb62f89b914633b26f9020eb06d647d87a/Spanner/src/Instance.php#L174)
Thats why we had to introduce `$options`